### DISCUSSION TITLE
Collapse navigation groups by default

### DISCUSSION THREAD

**Discussion by SebastiaanKloos** [filamentphp/filament#1368] [Ideas]
Hi all,

First of all, thank you very much @ryangjchandler for adding the collapsible navigation group feature to Filament. It already saves the state when navigating to other pages which is very cool.

I think being able to collapse a group by default would be awesome.

When I find the time, I can make a PR for that.

**Comment by danharrin** [comment]
I think it should be added to our new Advanced Navigation API (in the docs).
